flask/helpers.py

```diff
@@ -534,8 +535,13 @@ def send_file(filename_or_fp, mimetype=None, as_attachment=False,
         if attachment_filename is None:
             raise TypeError('filename unavailable, required for '
                             'sending as attachment')
+        filename_dict = {
+            'filename': (unicodedata.normalize('NFKD',
+                         text_type(attachment_filename)).encode('ascii',
+                                                                'ignore')),
+            'filename*': "UTF-8''%s" % url_quote(attachment_filename)}
```

Comment:
Just realized that even if we enhance `Headers.add`, we still can't use it because we can't guarantee that projects are using the latest Werkzeug version. 😞